tests/README.md

```diff
@@ -135,6 +135,9 @@ pipeline.
    export ACCESS_TOKEN_SECRET=<b>REPLACE_ME</b>
    export ACR_NAME=<b>REPLACE_ME</b>
    export AZURE_DEVOPS_EXT_PAT=<b>REPLACE_ME</b>
+   export SP_PASS=<b>REPLACE_ME</b>
```

Comment:
@mtarng Do one of the `spk` methods expect these as an env var? I was thinking we didn't needs these just to run the shells script

Reply:
https://github.com/CatalystCode/spk/blob/master/tests/validations.sh#L144

Seems like they're needed to create the variable groups, unless i'm missing something here.

Reply:
Got it. Thanks!